README.md

```diff
@@ -1,2 +1,63 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This is a plugin for Errbot (http://errbot.io) to manage a "topic-a-day" channel for daily discussion of topics
+
+## Installation
+To install Errbot, please use the manual [here](https://errbot.readthedocs.io/en/latest/user_guide/setup.html)
+
+## Usage
```

Comment:
Please make the usage section a link to https://errbot.readthedocs.io/en/latest/user_guide/administration.html#installing-plugins